src/main/webapp/dev/js/main/instructorFeedbacks.js

```diff
@@ -68,12 +70,12 @@ function formatDigit(num) {
 }
 
 /**
- * Format a date object into DD/MM/YYYY format
+ * Format a date object into D, dd M, yy format (Tue, 23 Jan, 2018)
```

Comment:
Seems inconsistent?

Reply:
@whipermr5 Shall I change the comment to `Format a date object into D, dd MMM, yyyy format. (Sat, 05 May, 2012)` ?

Reply:
I think follow the JavaScript format string. Method name too.